Psychrometrics_Engine/Compute/DensityWater.cs

```diff
@@ -29,18 +29,20 @@
 using BH.oM.Base.Attributes;
 using BH.oM.Base;
 using BH.Engine.Base;
+using BH.Engine.Units;
+using BH.oM.Quantities.Attributes;
 
 namespace BH.Engine.Psychrometrics
 {
     public static partial class Compute
     {
         [Description("Calculates water density from temperature.")]
-        [Input("temperature", "temperature (C).")]
-        [Output("density", "Density (kg/m3).")]
+        [Input("temperature", "Temperature.", typeof(Temperature))]
```

Comment:
Replace `temperature` with `banana` and...
![image](https://github.com/BHoM/Psychrometrics_Toolkit/assets/18049174/006bf0ff-64e4-47ee-a6fd-5d9bcaa50e59)

What should the temperature be of? Is there something extra we can provide the user to help them with this input?